docker/nfs-ganesha-ceph/nfs-ganesha-ceph-base/Dockerfile.j2

```diff
@@ -0,0 +1,30 @@
+FROM {{ namespace }}/{{ image_prefix }}openstack-base:{{ tag }}
+LABEL maintainer="{{ maintainer }}" name="{{ image_name }}"
+
+{% block nfs_ganesha_ceph_base_header %}{% endblock %}
+
+{% import "macros.j2" as macros with context %}
+
+{{ macros.configure_user(name='nfs-ganesha-ceph') }}
+
+{% if install_type == 'source' %}
```

Comment:
This template doesn't define a way to build if install_type is not `source`, but it seems like we just need to install some additional apt/deb packages, which would apply for install_type='binary' as well. We could just remove this if statement as a result, I think.